### ISSUE TITLE
False positive with PyQt5 "<class 'RuntimeError'>: wrapped C/C++ object of type QApplication has been deleted”

### ISSUE THREAD

**Issue by yku12cn** [PyCQA/pylint#3617]
### Steps to reproduce
1. Environment: VS-Code 1.45.0; Python 3.7.7; PyQt5; Pylint 2.5.0
2. Pylint settings:
```
    "python.linting.pylintEnabled": true,
    "python.linting.pycodestyleEnabled": true,
    "python.linting.enabled": true,
    "python.linting.pylintArgs": [
        "--extension-pkg-whitelist=PyQt5",
        "--generated-members=PyQt5.*",
        "--disable=invalid-name, protected-access, E1102"
    ],
```
3. check following code
```
    #!/usr/bin/env python3
    """
    A demo for pylint
    "wrapped C/C++ object of type QApplication has been deleted"
    false positive
    """
    import sys
    from PyQt5 import QtWidgets


    class Window(QtWidgets.QMainWindow):
        """A simple window"""
        def __init__(self):
            super(Window, self).__init__()
            self.variable = 1  # This line triggers the alarm.
            self.show()


    if __name__ == "__main__":
        app = QtWidgets.QApplication(sys.argv)
        newWindow = Window()
        sys.exit(app.exec_())

```

### Current behavior
report:
<class 'RuntimeError'>: wrapped C/C++ object of type QApplication has been deleted

more detailes:
[https://stackoverflow.com/questions/61775278/pylint-false-positive-class-runtimeerror-wrapped-c-c-object-of-type-qapp](url)

### Expected behavior
The code is totally fine. At least, Pylint 2.4.4 doesn't complain about anything.

### pylint --version output
pylint 2.5.0
astroid 2.4.0
Python 3.7.7 (default, May  6 2020, 11:45:54) [MSC v.1916 64 bit (AMD64)]

**Comment by PCManticore** [comment]
This has been fixed already with `astroid 2.4.1`. Please give it a go and reopen if it still does not work for you.

**Comment by yku12cn** [comment]
> This has been fixed already with `astroid 2.4.1`. Please give it a go and reopen if it still does not work for you.

Thanks, the fix is confirmed.
